fe/fe-core/src/main/java/org/apache/doris/catalog/EsTable.java

```diff
@@ -52,7 +52,9 @@
 @Getter
 @Setter
 public class EsTable extends Table implements GsonPostProcessable {
-    public static final Set<String> DEFAULT_DOCVALUE_DISABLED_FIELDS = new HashSet<>(Collections.singletonList("text"));
+    // refrence: https://www.elastic.co/guide/en/elasticsearch/reference/current/doc-values.html
+    public static final Set<String> DEFAULT_DOCVALUE_DISABLED_FIELDS =
+            new HashSet<>(Arrays.asList("text", "annotated_text"));
```

Comment:
What about match_only_text?

Reply:
`match_only_text`'s behavior is the same with `text` and should be included in here.